include/block/src/block/components/height-width.js

```diff
@@ -0,0 +1,36 @@
+/* global wp */
+const { TextControl, PanelBody, PanelRow } = wp.components;
+import { YesNoIcon } from './yes-no-icon';
+import '../editor.scss';
+
+export const HeightWidthSetting = ( props ) => {
+  const {
+    width,
+    onWidthChange,
+    WidthDashicon,
+    height,
+    onHeightChange,
+    HeightDashicon,
+  } = props;
+
+  return (
+    <PanelBody title="Player Width and Height Settings" className="vdo-height-width">
+      <PanelRow>
+        <TextControl
+          label="Player Width"
```

Comment:
https://wordpress.org/gutenberg/handbook/designers-developers/developers/components/text-control/#type

```suggestion
          label="Player Width"
          type="number"
```